cmd/maya-exporter/app/command/exporter_test.go

```diff
@@ -0,0 +1,64 @@
+package command
+
+import (
+	"net"
+	"reflect"
+	"testing"
+)
+
+func TestInitialize(t *testing.T) {
+	cases := map[string]struct {
+		cmdOptions *VolumeExporterOptions
+		output     string
+	}{
+		"Storage engine is cstor": {
+			cmdOptions: &VolumeExporterOptions{
+				StorageEngine: "cstor",
+			},
+			output: "cstor",
+		},
+		"storage engine is jiva": {
+			cmdOptions: &VolumeExporterOptions{
+				StorageEngine: "jiva",
+			},
+			output: "jiva",
+		},
+		"storage engine is other": {
+			cmdOptions: &VolumeExporterOptions{
+				StorageEngine: "other",
+			},
+			output: "",
+		},
+	}
+	for name, tt := range cases {
+		t.Run(name, func(t *testing.T) {
+			got := Initialize(tt.cmdOptions)
+			if !reflect.DeepEqual(got, tt.output) {
+				t.Fatalf("Initialize() => %v, want %v", got, tt.output)
```

Comment:
Can the fatal description be made more elaborate to be easily understood when this error happens.